docs/spec/spec-proposals/f1-fee-distribution/f1_fee_distr.tex

```diff
@@ -85,36 +105,61 @@ \subsection{Slashing}
 
 In practice this will not really be an efficiency hit, as we can expect most validators to have no slashes.
 Validators that get slashed a lot will naturally lose their delegators.
-A malicious validator that gets itself slashed many times would increase the gas to withdraw linearly, but the economic loss of funds due to the slashes should far out-weigh the extra overhead the withdrawer must pay for due to the gas.
+A malicious validator that gets itself slashed many times would increase the gas to withdraw linearly, but the economic loss of funds due to the slashes should far out-weigh the extra overhead the honest withdrawer must pay for due to the gas.
 
 \subsection{Inflation}
-Inflation is the idea that we want every staked coin to grow in value as time progresses. Each block, every staked token should each be rewarded $x$ staking tokens as inflation, where $x$ is calculated from function which takes state and the block information as input. This also allows for many seemless upgrade's to $x$'s algorithm. This can be added efficiently into the fee distribution model as follows:
+Inflation is the idea that we want every staked coin to create more staking tokens as time progresses. (To the purpose of driving down the worth of unstaked tokens) Each block, every staked token should produce $x$ staking tokens as inflation, where $x$ is calculated from a function $inflation$ which takes state and the block information as input. This inflation can be auto-bonded efficiently into the fee distribution model as follows:
```

Comment:
Hmm, I think the purpose is to pay network participants to stake (incurring some risk) and increase network security; decreasing the worth of unstaked tokens is a side effect, not the goal.

Reply:
This depends on who you talk to. @sunnya97 will argue for demurage being the optimal setting, where you decrease the tokens of the non-stakers :P